app/lib/pre_assembly/digital_object.rb

```diff
@@ -169,7 +169,8 @@ def build_from_staging_location(objects:, processing_configuration:, reading_ord
       FromStagingLocation::StructuralBuilder.build(cocina_dro: existing_cocina_object,
                                                    filesets:,
                                                    all_files_public: batch.batch_context.all_files_public?,
-                                                   reading_order:)
+                                                   reading_order:,
+                                                   manually_corrected_ocr: batch.batch_context.manually_corrected_ocr)
     end
```

Comment:
we need to know this new attribute in the class since it can affect how structural  is built